src/marine/hgodas/hgodas_adt2ioda.py

```diff
@@ -0,0 +1,105 @@
+#!/usr/bin/env python
```

Comment:
In the file src/marine/hgodas/CMakeLists.txt, the ${programs} list is not getting built properly. The files have underscores in their prefixes, whereas the CMakeLists.txt has hyphens.